src/en/space-station-14/departments/medical/proposals/Virology Reworked.md

```diff
@@ -0,0 +1,72 @@
+A successful vaccine will prevent the specific virus the vaccine is created for from jumping to any person who has not already been infected by the vaccine parent, but will not prevent any future mutation of the same virus eventually coming around and infecting later.
+
+## External Infection Sources
+Viruses that don't originate in the crew round start can be instead picked up from other sources, such as raw meats in the kitchen, not cleaning meat cutting tools in the kitchen, rats and mice, artifacts creating (usually very lethal) diseases, and sometimes external sources, such as carp bites, vent spider bites, spawned blood puddles, and potentially bio terror from nukkies.
+This is mostly done so the crew can't somehow systematically wipe out viruses.
+
+## Viral Modification
+Virologists can take multiple viruses and attempt to take characteristics from each of them in an effort to create new viruses, which can potentially be used to combat other viruses, or be turned into bioweappns.
+Viruses can be modified when exposed to Mutagen or Spacacillin to cause different effects.
+Mutagen will immediately trigger a mutation of the virus as would normally happen when the virus transfers between subjects. 
+Spacacillin can be used to "stabilize" the virus, and then later be used to intentionally infect people without the virus that the original sample was taken from. A Spacacillin modified virus would block the original virus from transferring to the patient as an alternative to a vaccine, as a form of inoculation instead but still subject the patient to the symptoms of the virus.
+
+## Contagion Tiers
+These can all stack in one virus based on volatility and mutation, and will always start with at least one of these.
+
+Blood to Blood: Requires direct blood contact to transfer, such as repeated syringe use, or the same melee weapon used against multiple people. Drinking blood may or may not count. Very low point cost.
+Saliva: Would require someone to drink or ingest the virus from an item someone else has drank or eaten off of without being cleaned first. Low point cost.
+Airborne, Inhalation: Would require people to stand near each other in order to transfer the virus. Can be blocked with breath masks. Moderate point cost.
+Contact: Virus can be picked up through contact with objects the infected has handled with a lifespan assigned to the virus for objects, as well as hugs, cuffing actions, buckling. Moderate point cost.
+Airborne, Infectious: High virulence, able to infect through anything via proximity except for full virology suits and the CMO hardsuit and Paramedic Softsuit (gotta make them useful for something). High point cost.
```

Comment:
Where would you think the janitor's biohazard suit ranks compared to the medical-specific ones? I'd imagine either on par or slightly worse.

Reply:
They should be equivalent IMO.